frontend/packages/helm-plugin/src/providers/useHelmChartRepositoriesBreadcrumbs.ts

```diff
@@ -0,0 +1,25 @@
+import { useTranslation } from 'react-i18next';
+import { match } from 'react-router';
+import { useActivePerspective } from '@console/dynamic-plugin-sdk';
+import { K8sKind } from '@console/internal/module/k8s';
+import { useTabbedTableBreadcrumbsFor } from '@console/shared';
+import { HelmChartRepositoryModel, ProjectHelmChartRepositoryModel } from '../models';
+import { helmPageTab } from '../utils/helm-tab-utils';
+
+export const getHelmChartRepositoriesModel = () => [
+  HelmChartRepositoryModel,
+  ProjectHelmChartRepositoryModel,
+];
+
+export const useHelmChartRepositoriesBreadcrumbs = (kindObj: K8sKind, urlMatch: match<any>) => {
+  const { t } = useTranslation();
+  const isAdminPerspective = useActivePerspective()[0] === 'admin';
+  return useTabbedTableBreadcrumbsFor(
+    kindObj,
+    urlMatch,
+    'helm-releases',
+    helmPageTab('Repositories'),
```

Comment:
This function was only called one time with `kind === 'Repositories'`. Any reason to not just use a string here?

```suggestion
    'repositories',
```